django/contrib/staticfiles/finders.py

```diff
@@ -26,11 +32,11 @@ def check(self, **kwargs):
             "configured correctly."
         )
 
-    def find(self, path, all=False):
+    def find(self, path, fetch_all=False, **kwargs):
```

Comment:
I'm not sold on the new proposed name, I think a more accurate option would be `match_all` or `find_all`. To me, fetch implies that something will be "brought" from there to here (ie like data retrieval or similar), which is not really the case.
What do you think?

Reply:
I agree, I think `find_all` it's a better fit for this param name. Handled in 704fdb92b513ee0d6218068990bd70afa90dfdfd